README.md

```diff
@@ -125,6 +125,46 @@ The standard `git diff` is not particularly useful for `.ipynb` files,
 especially for computed ones, but the rendered file can be viewed in PR
 and _nbdiff_ in command line can show a human-readable difference.
 
+### Adding Conda Environments to Jupyter Notebook
+
+ Conda allows you to create isolated environments with specific 
+ versions of Python and its packages. This prevents conflicts between 
+ different projects' dependencies.
+
+**Step 1: Create your conda environment.**
+Inside of your conda based command line use the following command
+`conda create --name firstEnv`
+firstEnv is the name of your environment. Change as needed.
+
+**Step 2: Activate your environment.**
+Using your newly created environment from the previous step, run
+`conda activate firstEnv`
+Now you have entered into your environment.
+
+**Step 3: Install any necessary packages for your environment.**
+Now you can install any package you need for your environment.
+Below is an example installation for pipenv, a dependency for
+popsborder.
+`conda install -c conda-forge pipenv`
```

Comment:
The documentation needs to be specific. The users need a specific set of instructions for the project. See how to install [pipenv dependencies in a conda environment](https://stackoverflow.com/a/51002659/574907) and _test it_.